resources/assets/lib/register-components.coffee

```diff
@@ -51,9 +51,8 @@ core.reactTurbolinks.register 'blockButton', (container) ->
 
 core.reactTurbolinks.register 'beatmap-discussion-events', (container) ->
```

Comment:
apparently I was converting this to typescript at some point and accidentally committed the unfinished file 🙃